src/app/dashboard/revalidation-history/revalidation-history.component.scss

```diff
@@ -0,0 +1,57 @@
+@import "~nhsuk-frontend/packages/core/settings/colours";
+
+.spacer {
+  flex: 1 1 auto;
+}
+
+table {
+  width: 100%;
```

Comment:
this can also probs be removed if you use the shared class

Reply:
done